tensorflow/core/tpu/tpu_initializer_helper.cc

```diff
@@ -99,11 +99,11 @@ bool IsTpuUsed(int64_t pid) {
 // processes owned by another user.
 // TODO (shahrokhi) use tensorflow/core/platform/filesystem (GetChildren) for
 // this.
-bool FindAndLogLibtpuProcess() {
+StatusOr<int64_t> FindAndLogLibtpuProcess() {
```

Comment:
Rename this `FindLibtpuProcess' and update the above comment, since it doesn't actually do any logging now.